NEWS

```diff
@@ -62,6 +62,11 @@ New Functionality
 * The DNS analyzer was extended to support TKEY RRs (RFC 2390). A corresponding
   ``dns_TKEY`` event was added.
 
+* The ``signature_match()`` and custom signature events receive the relative
+  end offset for the match within data as an optional parameter.
+
+	event signature_match(state: signature_state, msg: string, data: string, data_end_offset: count);
+
```

Comment:
Before I look more closely, let me double check that I understand the semantics correctly: is `data_end_offset` the offset of the final byte of the match inside *data*, or the offset of the final byte of `data` itself?

Reply:
The former and actually one behind the match. But yes, relative within the current `data` instance.

S for two packets with payloads
```
AAAAA
BXBBB
```
and `/.*X.*/` as signature, data would be populated with `BXBBB` and data_end_offset should be `2`.

The name isn't great. Maybe `data_match_size` instead?

(I'm also going to produce a pcap with above traffic to make sure this is 100% correct :-) )